website/templates/profile/account.mako

```diff
@@ -102,7 +104,7 @@
                         </form>
                     </div>
                 </div>
-				<div class="panel panel-default">
+<div class="panel panel-default">
```

Comment:
Very tiny nitpick, but could you make sure that the nesting of this div is appropriate?